<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_page</class>
 <widget class="QMainWindow" name="login_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>554</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>167772</width>
    <height>167772</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_Username">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_pb">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="deposit_rb">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Deposit </string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">transaction_group</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_accountb">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Account Balance: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accountusername">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>user name </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_balance">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Enter_pb">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Enter </string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_account_number">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>User's account number </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transaction">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Transaction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accountholder">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Account Holder :  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accountnumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Account number:  </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_input">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_enter_amount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Enter Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output_message">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="withdraw_rb">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bai Jamjuree</family>
     </font>
    </property>
    <property name="text">
     <string>Withdraw</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">transaction_group</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="transaction_group"/>
 </buttongroups>
</ui>
